system/telnet/telnet_client.c

```diff
@@ -115,13 +115,17 @@ static const struct telnet_telopt_s g_telopts[] =
 
 static void send_local_input(char *buffer, int size)
```

Comment:
Maybe for the next PR:
```suggestion
static void send_local_input(FAR char *buffer, int size)
```

Reply:
the whole code base of telnet doesn't add FAR before pointer, so let's keep as before.